frontend/components/card/discussion/CardDiscussionInput.vue

```diff
@@ -83,27 +83,41 @@
           />
         </div>
       </div>
-      <div class="w-full md:w-full">
+      <div v-if="discussionInput.highRisk" class="w-full md:w-full">
         <textarea
           id="message"
           rows="4"
-          class="block p-2.5 w-full text-sm text-light-text bg-light-content rounded-lg border border-light-section-div dark:bg-dark-content dark:border-dark-section-div placeholder-light-special-text dark:placeholder-dark-special-text dark:text-dark-text focus-brand"
-          :placeholder="$t('components.card-discussion-input.leave-comment')"
+          class="block p-2.5 w-full text-sm text-light-text rounded-lg border border-red-500 placeholder-light-special-text focus-brand  dark:bg-dark-distinct font-bold red-text"
```

Comment:
Looking at this a bit here all of you: for one thing it'd be good if we got rid of the `<style>` block and just used Tailwind here. We'd want this class to include `text-light-action-red dark:text-dark-action-red` instead of `red-text` once that's removed. We can also do the same for `border-light-action-red dark:border-dark-action-red` instead of `border-red-500` :)

Reply:
We also need to add `dark:text-dark-text` back into it as we currently have the text as black for dark mode in the high risk version of the field :)

Reply:
And the backgrounds have been changed, which actually was correct given the designs even though the code had it the way we wanted it. This is my fault 😅 We'd need `bg-light-content dark:bg-dark-content` back in each of them :)

Reply:
```suggestion
          class="block p-2.5 w-full text-sm rounded-lg border border-light-action-red dark:border-dark-action-red placeholder-light-special-text focus-brand  dark:bg-dark-distinct font-bold text-light-action-red dark:text-dark-action-red"
```

I have made these changes locally however, it does not display the text in red but rather the inputted text is displayed in red. This is why we added the style tag, so i'm still unsure how to make that work

Reply:
> And the backgrounds have been changed, which actually was correct given the designs even though the code had it the way we wanted it. This is my fault 😅 We'd need `bg-light-content dark:bg-dark-content` back in each of them :)

does this mean I should replace dark:bg-dark-distinct?

Reply:
Yes please replace `dark:bg-dark-distinct`:)

Reply:
And sorry, for the text we should do `placeholder:` before it as seen in the [Tailwind docs](https://tailwindcss.com/docs/hover-focus-and-other-states#placeholder-text). It should be `text-light-text dark:text-dark-text placeholder:text-light-action-red dark:placeholder:text-dark-action-red` :)

Reply:
Let me know if this works, @mahasultan!

Reply:
Yes, works ! thank you

Reply:
Welcome! Thank you!